content/en/docs/contributing/release-process.md

```diff
@@ -12,16 +12,25 @@ new release of cert-manager.
 
 First ensure that you have all the tools and permissions required to perform a cert-manager release:
 
-1. Install [Kubernetes Release Notes Generator](https://github.com/kubernetes/release/blob/master/cmd/release-notes/README.md).
-2. Install [cert-manager release tooling](https://github.com/cert-manager/release).
-3. Get permission to use the "cert-manager-release" project in Google Cloud Platform.
+1. Install the [`release-notes`]((https://github.com/kubernetes/release/blob/master/cmd/release-notes/README.md)) CLI:
+   ```sh
+   (cd && GO111MODULE=on go get k8s.io/release/cmd/release-notes@latest)
+   ```
+2. Install our [`cmrel`](https://github.com/cert-manager/release) CLI:
+   ```sh
+   (cd && GO111MODULE=on go get github.com/cert-manager/release/cmd/cmrel@latest)
+   ```
+3. Get the "Cloud Build Editor" permission for the
```

Comment:
Not so important, but perhaps it's also worth to get view permissions for Storage- in case there is a need to manually check the releases bucket.